Automattic/wp-calypso
# Gifting: Remove site url param on gifting checkout loggedOut redirect

When not logged in and trying to purchase a site Gift, it redirects to login using Subscription ID as a Site ID.

#### Proposed Changes

Remove site URL param when redirecting not logged in users only when on gifting checkout.

#### Testing Instructions

- Follow [these instructions](pdtkmj-Ks-p2) to test the Gifting subscription
- Try to make a Gift when not logged in.
- Make sure you don't see the `or visit xxx` under `Log in to your account`

#### Screenshots
| Before | After |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/402286/203854916-427ff5b8-8665-4081-8ab2-b323ac7abe48.png) | ![image](https://user-images.githubusercontent.com/402286/203854926-4a6ade47-d8c4-4173-aadf-aed52d675336.png) |

#### Pre-merge Checklist




Fixed https://github.com/Automattic/dotcom-forge/issues/1292
